### PR TITLE
Revert ALP content back to TSEE version 2.5.

### PR DESCRIPTION
**Pull request by bmckercher123**
KP noted current version of "Application layer policy tutorial" in Calico Enterprise was using "cumbersome" commands (aka calicoctl) and that correct version was in 2.5. Spike verified that 2.5 is correct version of commands, but said we should revalidate the steps and **fix/merge to OS first**. 

- Dev to validate steps 
- Merge to nightly and cherry pick to /master
- Merge to private and cherry pick to /master

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit bc1fb6972ddaeea47c6de9e5adc4b128b87133fa

https://deploy-preview-3366--calico.netlify.com

File: security/tutorials/app-layer-policy/enforce-policy-istio.md [hunk 1]
```diff
@@ -159,33 +201,44 @@ If you still have your shell open in the customer pod, exit out or open a new te
...
 
-We can mitigate both of the above deficiencies with a {{site.prodname}} policy.
+We can mitigate both of the above deficiencies with a {{site.prodname}} policy (if you still have your shell open in 
+the attack pod, first exit out or open a new terminal tab)
 
-    wget {{ "/security/tutorials/app-layer-policy/manifests/30-policy.yaml" | absolute_url }}
-    calicoctl create -f 30-policy.yaml
+```bash
+wget {{ "/security/tutorials/app-layer-policy/manifests/30-policy.yaml" | absolute_url }}
+kubectl create -n yaobank -f 30-policy.yaml
```
Comment (gianlucam76) [inline security/tutorials/app-layer-policy/enforce-policy-istio.md:240]: This does not work. Resource in 30-policy.yaml are 
apiVersion: projectcalico.org/v3
kind: GlobalNetworkPolicy

If we want to use kubectl, those should be


apiVersion: crd.projectcalico.org/v1
kind: GlobalNetworkPolicy

File: security/tutorials/app-layer-policy/enforce-policy-istio.md [hunk 2]
```diff
@@ -196,10 +249,11 @@ microservice.
 
 ```yaml
 apiVersion: projectcalico.org/v3
-kind: GlobalNetworkPolicy
+kind: NetworkPolicy
```
Comment (gianlucam76) [inline security/tutorials/app-layer-policy/enforce-policy-istio.md:252]: The policy in the yaml used above are GlobalNetworkPolcy, so not sure why we are changing this to NetworkPolicy.

File: security/tutorials/app-layer-policy/enforce-policy-istio.md [hunk 3]
```diff
@@ -216,10 +270,11 @@ requests.
...
```
Comment (gianlucam76) [inline security/tutorials/app-layer-policy/enforce-policy-istio.md:273]: The policy in the yaml used above are GlobalNetworkPolcy, so not sure why we are changing this to NetworkPolicy.

File: security/tutorials/app-layer-policy/enforce-policy-istio.md [hunk 4]
```diff
@@ -237,10 +292,11 @@ account for the customer web app.
...
 metadata:
-  name: database
+  name: default.database
```
Comment (gianlucam76) [inline security/tutorials/app-layer-policy/enforce-policy-istio.md:297]: The policy in the yaml used above are GlobalNetworkPolcy, so not sure why we are changing this to NetworkPolicy.

File: security/tutorials/app-layer-policy/enforce-policy-istio.md [hunk 5]
```diff
@@ -82,16 +91,41 @@ summary-token-8kpt1    kubernetes.io/service-account-token   3         21h
...
+   ```
+   {: .no-select-button}
+
+   The address of the ingressgateway service is the external IP of the `istio-ingressgateway`, followed by port 80:
+
+   ```bash
+   export GATEWAY_URL=130.211.10.121:80
+   ```
+
+1. If your cluster does not support external load balancers, you can use the public IP of the worker
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply security/tutorials/app-layer-policy/enforce-policy-istio.md]: fixed

File: security/tutorials/app-layer-policy/enforce-policy-istio.md (hunk 2 above)
Comment (bmckercher123) [reply security/tutorials/app-layer-policy/enforce-policy-istio.md]: fixed

File: security/tutorials/app-layer-policy/enforce-policy-istio.md (hunk 3 above)
Comment (bmckercher123) [reply security/tutorials/app-layer-policy/enforce-policy-istio.md]: fixed.

File: security/tutorials/app-layer-policy/enforce-policy-istio.md (hunk 4 above)
Comment (bmckercher123) [reply security/tutorials/app-layer-policy/enforce-policy-istio.md:240]: fixed.

**Comment by bmckercher123** [comment]
Brendan approves making both OS and private versions reflect kdd. That said, how do we change this file? Do we need one for etcd for legacy versions, and new one 3.0/3.14+? 

Also, what page can I link to kdd?

**Comment by bmckercher123** [comment]
This PR was started based on 2.5 version noted by Spike. However, both OS and Enterprise docs were updated in the meantime to reflect reality (which I was not aware of). Verified with Turk, and the only thing that needs to change is to delete reference to calicoctl requirement in Enterprise doc. Closing this doc and opening separate doc to address the small issue.

**Pull request closed without merging**